src/StripeTests/NullableValueTypes.cs

```diff
@@ -0,0 +1,101 @@
+namespace StripeTests
+{
+    using System;
+    using System.Collections;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Reflection;
+
+    using Microsoft.Extensions.DependencyModel;
+    using Newtonsoft.Json;
+    using Stripe;
+    using Xunit;
+
+#if NETCOREAPP1_1
+    public class NullableValueTypes
+    {
+        [Fact]
+        public void EnsureAllValueTypesAreNullable()
+        {
+            List<string> results = new List<string>();
+
+            // Get all classes that implement INestedOptions
+            var type = typeof(INestedOptions);
```

Comment:
`BaseOptions` actually implements `INestedOptions` -- I changed this in #1276. We should probably rename `INestedOptions` to e.g. `IOptions` since it now simply indicates that a class contains request parameters, regardless of whether the parameters are nested or not.

Reply:
Oh I did miss that. Renaming would definitely make sense though I wonder if having an Interface for nested options is a good way to separate top-level options versus others